data/json/recipes/recipe_deconstruction.json

```diff
@@ -1329,9 +1276,8 @@
             {"id":"SCREW","level":1,"amount":1},
             {"id":"WRENCH","level":1,"amount":1},
             {"id":"SAW_M","level":1,"amount":1}
-        ], "tools": [
-            [ [ "soldering_iron", 40 ], [ "toolset", 40 ] ],
-            [ [ "oxy_torch", 6 ], [ "welder", 30 ], [ "toolset", 45 ], [ "welder_crude", 45 ] ]
+        ], "tools": [ [ [ "soldering_iron", 40 ], [ "toolset", 40 ] ],
```

Comment:
@karols146 I merged current PR because it fixes the bug introduced previously but formatting here still not correct.

* This is how it's formatted now:
```
], "tools": [ [ [ "soldering_iron", 40 ], [ "toolset", 40 ] ],
            [ [ "oxy_torch", 6 ], [ "welder", 30 ], [ "toolset", 45 ], [ "welder_crude", 45 ] ]
], "components": [
```

* This is a more correct version of formatting:
```
], "tools": [ [ [ "soldering_iron", 40 ], [ "toolset", 40 ] ],
              [ [ "oxy_torch", 6 ], [ "welder", 30 ], [ "toolset", 45 ], [ "welder_crude", 45 ] ]
], "components": [
```
* or even better (at least for me):
```
], "tools": [ 
              [ [ "soldering_iron", 40 ], [ "toolset", 40 ] ],
              [ [ "oxy_torch", 6 ], [ "welder", 30 ], [ "toolset", 45 ], [ "welder_crude", 45 ] ]
], "components": [
```
Note that first ```[``` embraces all sets and it's not correct to use identical indent for first & second lines.